dotcom-rendering/scripts/webpack/server.js

```diff
@@ -18,6 +18,12 @@ module.exports = () => ({
 		require('webpack-node-externals')({
 			allowlist: [/^@guardian/],
 		}),
+		({ request }, callback) => {
+			return process.env.NODE_ENV === 'development' &&
+				/^(@aws-.+\/).*/i.test(request)
```

Comment:
nitpick: Could we use 
```suggestion
				request.startsWith('@aws-sdk')
```

to avoid having to speak RegExp?

Reply:
Not a nitpick at all - it's a very good point! Thanks.